crates/yttrium/src/chain_abstraction/client.rs

```diff
@@ -490,16 +494,24 @@ impl Client {
 }
 
 #[derive(Debug)]
+// #[cfg_attr(feature = "uniffi", derive(uniffi_macros::Record))]
```

Comment:
Uncomment?

Reply:
It can't be derived currently because `Eip1559Estimation` (inside `TxnDetails`) doesn't implement UniFFI tag, but in the future we can so I left it here as a placeholder